### ISSUE TITLE
[vite] Internal server error: Missing "./src/lib/custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package

### ISSUE THREAD

**Issue by jcalebsmith** [bmschmidt/pubmed-explorer#65]
I've installed everything per the readme and have verified that I can get deepscatter working, however, once I run 'npm run dev' and attempt to go to http://localhost:2001/pubmed, I get an error stating 
```
5:38:37 PM [vite] Internal server error: Missing "./src/lib/custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package
  Plugin: vite:import-analysis
  File: F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte
```
Below I've pasted the full output. I'm not sure what the issue is. I can run deepscatter just fine on its own. I've verified that pandoc is installed and is being found without issue. 

```
(pubmed-explorer) F:\Test\Projects\pubmed-explorer>npm run dev

> scrollershots@0.0.2 dev
> vite dev --port 2001 --host

  VITE v4.4.9  ready in 1051 ms

  ➜  Local:   http://localhost:2001/
  ➜  Network: http://192.168.0.112:2001/
  ➜  press h to show help
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
5:38:37 PM [vite] Internal server error: Missing "./src/lib/custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package
  Plugin: vite:import-analysis
  File: F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte
      at e (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:25)
      at n (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:627)
      at o (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:1297)
      at resolveExportsOrImports (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28712:20)
      at resolveDeepImport (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28731:31)
      at tryNodeResolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28419:20)
      at Context.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28180:28)
      at async Object.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:44207:32)
      at async TransformContext.resolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:43923:23)
      at async normalizeUrl (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:41793:34)
Missing "./src/lib/custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/CallMethod.svelte:14:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/FancyDivs.svelte:2:11 FancyDivs has unused export property 'data'. If it is for external reference only, please consider using `export const data`
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SpecialCode.svelte:3:12 SpecialCode has unused export property 'data'. If it is for external reference only, please consider using `export const data`
Missing "./custom/ScrollershipCodeBlock.svelte" specifier in "pandoc-svelte-components" package
Failed to load url /src/lib/hammingsearch (resolved id: F:/Test/Projects/pubmed-explorer/src/lib/hammingsearch) in F:/Test/Projects/pubmed-explorer/src/lib/components/SpecialCode.svelte. Does the file exist?
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Raw-Slider.svelte:6:11 Raw_Slider has unused export property 'step'. If it is for external reference only, please consider using `export const step`
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Search.svelte:201:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
Missing "./Div.svelte" specifier in "pandoc-svelte-components" package
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Dropdown.svelte:55:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Controls.svelte:69:1 Unused CSS selector "#holder"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/pubmed/SelectIds.svelte:98:10 Security: Anchor with "target=_blank" should have rel attribute containing the value "noreferrer"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/pubmed/SelectIds.svelte:1:29 SelectIds has unused export property 'data'. If it is for external reference only, please consider using `export const data`
Failed to load url /src/lib/wasm_regex/wasm_regex (resolved id: F:/Test/Projects/pubmed-explorer/src/lib/wasm_regex/wasm_regex) in F:/Test/Projects/pubmed-explorer/src/lib/stringsearch.ts. Does the file exist?
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/Legend.svelte:12:4 $: has no effect outside of the top-level
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/Legend.svelte:40:1 Unused CSS selector ".quantlegend"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SearchResults.svelte:81:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SearchResults.svelte:87:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:58:1 Unused CSS selector ".axis text"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:61:1 Unused CSS selector ".axis path"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:62:1 Unused CSS selector ".axis line"
5:38:37 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:67:1 Unused CSS selector ".axis.tick"
5:39:07 PM [vite] page reload src/routes/pubmed/+page.svelte
Internal server error: Missing "./custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package
  Plugin: vite:import-analysis
  File: F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte
      at e (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:25)
      at n (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:627)
      at o (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:1297)
      at resolveExportsOrImports (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28712:20)
      at resolveDeepImport (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28731:31)
      at tryNodeResolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28419:20)
      at Context.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28180:28)
      at Object.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:44207:64)
      at async TransformContext.resolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:43923:23)
      at async normalizeUrl (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:41793:34)
Error: Missing "./custom/ScrollershipWrapper.svelte" specifier in "pandoc-svelte-components" package
    at e (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:25)
    at n (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:627)
    at o (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:21420:1297)
    at resolveExportsOrImports (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28712:20)
    at resolveDeepImport (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28731:31)
    at tryNodeResolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28419:20)
    at Context.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:28180:28)
    at Object.resolveId (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:44207:64)
    at async TransformContext.resolve (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:43923:23)
    at async normalizeUrl (file:///F:/Test/Projects/pubmed-explorer/node_modules/vite/dist/node/chunks/dep-df561101.js:41793:34) {
  plugin: 'vite:import-analysis',
  id: 'F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte',
  pluginCode: '/* F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte generated by Svelte v3.55.0 */\n' +
    'import {\n' +
    '\tcreate_ssr_component,\n' +
    '\tescape,\n' +
    '\tsubscribe,\n' +
    '\tvalidate_component,\n' +
    '\tvalidate_store\n' +
    '} from "svelte/internal";\n' +
    '\n' +
    "import API from '$lib/deepscatter.js';\n" +
    "import Scrollership from 'pandoc-svelte-components/custom/ScrollershipWrapper.svelte';\n" +
    "import { page } from '$app/stores';\n" +
    "import { base } from '$app/paths';\n" +
    "import CallMethod from '$lib/components/CallMethod.svelte';\n" +
    "import Dropdown from '$lib/components/Dropdown.svelte';\n" +
    "import Search from '$lib/components/Search.svelte';\n" +
    "import Slider from '$lib/components/Slider.svelte';\n" +
    "import Legend from '$lib/components/ColorLegend.svelte';\n" +
    "import SpecialCode from '$lib/components/SpecialCode.svelte';\n" +
    "import FancyDivs from '$lib/components/FancyDivs.svelte';\n" +
    "import Controls from '$lib/components/Controls.svelte';\n" +
    "import ScrollershipCodeBlockWrapper from '$lib/components/ScrollershipCodeBlockWrapper.svelte';\n" +
    "import RangeSlider from '$lib/components/RangeSlider.svelte';\n" +
    "import AboutModal from '$lib/components/Modal.svelte';\n" +
    "import PubmedIdSearch from '$lib/components/pubmed/SelectIds.svelte';\n" +
    '\n' +
    'const css = {\n' +
    '\tcode: ".shrink.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{flex-grow:2}.title.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{flex-grow:4;margin:auto;font-size:24pt}.about-icon.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{display:flex;justify-content:center;align-items:center;width:32px;height:32px;border-radius:50%;background-color:#ffff;transition:background-color 0.2s}.about-icon.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr:hover{background-color:#222222ff}.about-icon.s-lrXS01zkuZKr span.s-lrXS01zkuZKr.s-lrXS01zkuZKr{font-size:24px;font-weight:bold;color:#222222ff;transition:color 0.2s}.about-icon.s-lrXS01zkuZKr:hover span.s-lrXS01zkuZKr.s-lrXS01zkuZKr{color:#ffff}.header.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{display:flex;flex-direction:row;align-items:center;justify-content:center}.logos.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{display:flex;flex-direction:row}.logo.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{max-height:48px}.logo.s-lrXS01zkuZKr>a.s-lrXS01zkuZKr>img.s-lrXS01zkuZKr{max-height:48px}.s-lrXS01zkuZKr.s-lrXS01zkuZKr.s-lrXS01zkuZKr{}",\n' +
    '\tmap: null\n' +
    '};\n' +
    '\n' +
    'const Page = create_ssr_component(($$result, $$props, $$bindings, slots) => {\n' +
    '\tlet $page, $$unsubscribe_page;\n' +
    "\tvalidate_store(page, 'page');\n" +
    '\t$$unsubscribe_page = subscribe(page, value => $page = value);\n' +
    '\tlet { data } = $$props;\n' +
    '\tconst { document } = data;\n' +
    '\n' +
    '\tconst controls = {\n' +
    "\t\t'codeblock.method': CallMethod,\n" +
    "\t\t'codeblock.dropdown': Dropdown,\n" +
    "\t\t'codeblock.search': Search,\n" +
    "\t\t'codeblock.callmethod': CallMethod,\n" +
    "\t\t'codeblock.slider': Slider,\n" +
    "\t\t'codeblock.double-slider': RangeSlider,\n" +
    "\t\t'codeblock.api': ScrollershipCodeBlockWrapper,\n" +
    "\t\t'div.pmidsearch': PubmedIdSearch,\n" +
    "\t\t'div.legend': Legend,\n" +
    "\t\t'div.barchartprep': SpecialCode,\n" +
    "\t\t'div.fancydiv': FancyDivs\n" +
    '\t};\n' +
    '\n' +
    '\tconst { slug } = $page.params;\n' +
    '\tlet promiser = null;\n' +
    '\n' +
    '\tconst scatterpromise = new Promise(resolve => {\n' +
    '\t\t\tpromiser = resolve;\n' +
    '\t\t});\n' +
    '\n' +
    '\tlet about_modal = false;\n' +
    '\n' +
    '\tfunction toggleModal() {\n' +
    '\t\tabout_modal = !about_modal;\n' +
    '\t}\n' +
    '\n' +
    '\tfunction closeModal() {\n' +
    '\t\tabout_modal = false;\n' +
    '\t}\n' +
    '\n' +
    '\tif ($$props.data === void 0 && $$bindings.data && data !== void 0) $$bindings.data(data);\n' +
    '\t$$result.css.add(css);\n' +
    '\t$$unsubscribe_page();\n' +
    '\n' +
    '\treturn `<div id="${"scrollershipwrapper"}" class="${"-mt-4 s-lrXS01zkuZKr"}">${validate_component(Scrollership, "Scrollership").$$render(\n' +
    '\t\t$$result,\n' +
    '\t\t{\n' +
    '\t\t\tcontrols,\n' +
    '\t\t\tAPI,\n' +
    '\t\t\tast: document,\n' +
    "\t\t\tposition: 'left',\n" +
    '\t\t\tpromiser\n' +
    '\t\t},\n' +
    '\t\t{},\n' +
    '\t\t{\n' +
    '\t\t\t"control-items": () => {\n' +
    '\t\t\t\treturn `<div slot="${"control-items"}" id="${"controls"}" class="${"pointer-events-none s-lrXS01zkuZKr"}">${validate_component(Controls, "Controls").$$render($$result, { scatterpromise }, {}, {})}</div>`;\n' +
    '\t\t\t},\n' +
    '\t\t\t"custom-navbar": () => {\n' +
    '\t\t\t\treturn `<div slot="${"custom-navbar"}" class="${"header s-lrXS01zkuZKr"}"><div class="${"shrink s-lrXS01zkuZKr"}"></div>\r\n' +
    '\t\t\t<div class="${"title s-lrXS01zkuZKr"}">${escape(document.meta.title.c.map(d => d.c).join(\' \'))}</div>\r\n' +
    '\t\t\t<div class="${"logos s-lrXS01zkuZKr"}"><div class="${"logo s-lrXS01zkuZKr"}"><a href="${"https://www.eye-tuebingen.de/berenslab/"}" class="${"s-lrXS01zkuZKr"}"><img alt="${"Tübingen University logo"}" src="${"UniversitaetTuebingen_WortBildMarke.png"}" class="${"s-lrXS01zkuZKr"}"></a></div>\r\n' +
    '\t\t\t\t<div class="${"logo mx-8 s-lrXS01zkuZKr"}"><a href="${"http://nomic.ai/"}" class="${"s-lrXS01zkuZKr"}"><img src="${"nomic_with_transparent.png"}" alt="${"Nomic logo"}" class="${"s-lrXS01zkuZKr"}"></a></div></div>\r\n' +
    '\r\n' +
    '\t\t\t<div id="${"about"}" class="${"s-lrXS01zkuZKr"}">\r\n' +
    '\t\t\t\t<button class="${"btn btn-primary s-lrXS01zkuZKr"}" data-bs-toggle="${"modal"}" data-bs-target="${"#aboutModal"}"><div class="${"about-icon s-lrXS01zkuZKr"}"><span class="${"s-lrXS01zkuZKr"}">i</span></div></button></div></div>`;\n' +
    '\t\t\t},\n' +
    '\t\t\tdefault: () => {\n' +
    '\t\t\t\treturn `${about_modal\n' +
    '\t\t\t\t? `${validate_component(AboutModal, "AboutModal").$$render($$result, {}, {}, {})}`\n' +
    '\t\t\t\t: ``}`;\n' +
    '\t\t\t}\n' +
    '\t\t}\n' +
    '\t)}\r\n' +
    '</div>`;\n' +
    '});\n' +
    '\n' +
    'export default Page;\n' +
    'import "F:/Test/Projects/pubmed-explorer/src/routes/pubmed/+page.svelte?svelte&type=style&lang.css";\n'
}
5:39:07 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/CallMethod.svelte:14:0 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:39:07 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SpecialCode.svelte:3:12 SpecialCode has unused export property 'data'. If it is for external reference only, please consider using `export const data`
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/FancyDivs.svelte:2:11 FancyDivs has unused export property 'data'. If it is for external reference only, please consider using `export const data`
Missing "./custom/ScrollershipCodeBlock.svelte" specifier in "pandoc-svelte-components" package
Failed to load url /src/lib/hammingsearch (resolved id: F:/Test/Projects/pubmed-explorer/src/lib/hammingsearch) in F:/Test/Projects/pubmed-explorer/src/lib/components/SpecialCode.svelte. Does the file exist?
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Raw-Slider.svelte:6:11 Raw_Slider has unused export property 'step'. If it is for external reference only, please consider using `export const step`
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Dropdown.svelte:55:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Search.svelte:201:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
Missing "./Div.svelte" specifier in "pandoc-svelte-components" package
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/Controls.svelte:69:1 Unused CSS selector "#holder"
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/pubmed/SelectIds.svelte:98:10 Security: Anchor with "target=_blank" should have rel attribute containing the value "noreferrer"
5:39:08 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/pubmed/SelectIds.svelte:1:29 SelectIds has unused export property 'data'. If it is for external reference only, please consider using `export const data`
Failed to load url /src/lib/wasm_regex/wasm_regex (resolved id: F:/Test/Projects/pubmed-explorer/src/lib/wasm_regex/wasm_regex) in F:/Test/Projects/pubmed-explorer/src/lib/stringsearch.ts. Does the file exist?
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/Legend.svelte:12:4 $: has no effect outside of the top-level
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/Legend.svelte:40:1 Unused CSS selector ".quantlegend"
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SearchResults.svelte:81:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/SearchResults.svelte:87:4 A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:58:1 Unused CSS selector ".axis text"
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:61:1 Unused CSS selector ".axis path"
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:62:1 Unused CSS selector ".axis line"
5:39:09 PM [vite-plugin-svelte] F:/Test/Projects/pubmed-explorer/src/lib/components/deepscatter/d3Legend.svelte:67:1 Unused CSS selector ".axis.tick"
```

**Comment by bmschmidt** [comment]
Oh dear. It looks like I've messed up the npm packaging for `pandoc-svelte-components`. I will try to fix this this week.

But be aware, I'm not sure it's possible now to run this library directly. There are a few other issues that keep it from being served locally (including having the 20 million items served locally)

**Comment by jcalebsmith** [comment]
> Oh dear. It looks like I've messed up the npm packaging for `pandoc-svelte-components`. I will try to fix this this week.
> 
> But be aware, I'm not sure it's possible now to run this library directly. There are a few other issues that keep it from being served locally (including having the 20 million items served locally)

No worries at all! Deepscatter works just fine, which is the main thing. 

What I was hoping to do was use this repo as a template/guide for my own project. In my project, we've clustered the pubmed corpus into ~30,000 nodes and have assigned x,y coordinates using our own algorithm. I'd like to use deepscatter for serving this visualization (and would love to figure out a hack to get it to incorporate edges!). My interest in this repo was primarily to use it as a UI and storytelling template. I love the scroller narrative you incorporated, and the UI is very clean and intuitive. I'm no web developer though, so I was hoping to plug in my own point data and then tweak the UI, scroller narrative, etc. as necessary rather than trying to recreate what you've already done. 

I'd be grateful for any advice you have for incorporating UI and storytelling aspects similar to what you've done here top of Deepscatter! 

tl;dr I love deepscatter in general and the UI you've put together here in particular. However, I need to substitute my own point data (~30,000 points). All other functionality remains the same.

**Comment by dkobak** [comment]
Hi Caleb! I guess I know what dataset you are talking about ;-) (https://www.nature.com/articles/s41597-020-00749-y) Would indeed be great if there were an explorable visualisation of it available, I'm looking forward! Dmitry

**Comment by jcalebsmith** [comment]
> Hi Caleb! I guess I know what dataset you are talking about ;-) (https://www.nature.com/articles/s41597-020-00749-y) Would indeed be great if there were an explorable visualisation of it available, I'm looking forward! Dmitry

Hi Dmitry! I've been found out! Congratulations on this paper (and thank you for posting your scripts on github)! It's certainly creating a buzz and the visualizations are beautiful.

We've just finished a new version of our own model and I would like to get something up so people can interact with it. I would love to see what the overlap is between your embedding-based pubmed clusters and our citation-based clusters. I have the beginnings of an idea for how the two might be combined...I'll reach out over email.
